app/main.py

```diff
@@ -1,3 +1,47 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
```

Comment:
`km` could be not only `float`, but `int` as well. Change your type annotation here and in the other methods to encounter that.